### ISSUE TITLE
Publish .NET Standard-compatible Nuget package

### ISSUE THREAD

**Issue by adriangodong** [boonzaai/GremlinQuery.net#3]
I'm trying to use this package with other .NET Standard packages. Please rebuild and republish this as .NET Standard package.

You can start with the following snippet for `Icris.GremlinQuery.csproj`:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <Title>Icris.GremlinQuery</Title>
    <PackageId>GremlinQueryPackage</PackageId>
    <IncludeSymbols>True</IncludeSymbols>
    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
    <Version>0.0.0</Version>
    <PackageProjectUrl>https://github.com/boonzaai/GremlinQuery.net</PackageProjectUrl>
    <PackageLicenseUrl>https://github.com/boonzaai/GremlinQuery.net/blob/master/LICENSE</PackageLicenseUrl>
  </PropertyGroup>

  <PropertyGroup>
    <TargetFramework>netstandard1.0</TargetFramework>
    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
  </PropertyGroup>

</Project>
```

I can help you with migration and build setup if needed.

**Comment by boonzaai** [comment]
Hi Adrian,

I’ve updated the package using the versions you requested, and published it under version 0.3.0. Github repo is also updated.

Best regards,
André

**Comment by adriangodong** [comment]
Awesome! I've pulled the new version and works as expected.

Thank you for the quick turn around.
